release-tools/build.make

```diff
@@ -101,9 +106,42 @@ push-%: container-%
 		fi; \
 	done
 
+RELEASE_ALIAS_TAG=$(PULL_BASE_REF)
+
+push-multiarch-%:
+	export DOCKER_CLI_EXPERIMENTAL=enabled
+	make BUILD_PLATFORMS="windows amd64 .exe"
+	gcloud auth configure-docker
+	docker buildx create --use --name multiarchimage-buildertest
+	set -ex; \
+	pushMultiArch () { \
+                tag=$$1  ;\
+                docker buildx build --push -t $(MULTIARCH_IMAGE_NAME):amd64-linux-$$tag --platform=linux/amd64 -f Dockerfile.multiarch . ;\
+                docker buildx build --push -t $(MULTIARCH_IMAGE_NAME):s390x-linux-$$tag --platform=linux/s390x -f Dockerfile.multiarch . ;\
+                docker buildx build --push -t $(MULTIARCH_IMAGE_NAME):amd64-windows-$$tag --platform=windows -f Dockerfile.Windows . ;\
+                docker manifest create --amend $(MULTIARCH_IMAGE_NAME):$$tag $(MULTIARCH_IMAGE_NAME):amd64-linux-$$tag \
+                        $(MULTIARCH_IMAGE_NAME):s390x-linux-$$tag \
+                        $(MULTIARCH_IMAGE_NAME):amd64-windows-$$tag ;\
+                docker manifest push -p $(MULTIARCH_IMAGE_NAME):$$tag ;\
+	}; \
+	if [ $(RELEASE_ALIAS_TAG) = "master" ]; then \
+                       : "creating or overwriting canary image"; \
+                       pushMultiArch canary ; \
```

Comment:
For canary images, the `rev` label used to specify the source code revision (= result of git describe). This is useful to determine what the image contains because there is no other indicator.

Can this be done also for cloud builds? If we override `REV` in `cloudbuild.yaml`, then we can simply use `--label rev=$(REV)` here.

Reply:
As `rev` value from git describe command(build.make) is same as `tag-n-hash` value from _GIT_TAG( vYYYYMMDD-tag-n-ghash form). Added code to extract `tag-n-hash` value from REV(REV_CI cloudbuild env)in push-multiarch target and passing this as label value to docker build.